src/widgets/peoplePicker.ts

```diff
@@ -379,9 +402,9 @@ export class GroupBuilder {
     })
   }
 
-  handleRemove (webIdNode) {
+  handleRemove (webIdNode: NamedNode) {
     return _event => {
-      const statement = rdf.st(this.group, ns.vcard('hasMember'), webIdNode)
```

Comment:
any suggestions on how to handle this?  rdf.st needs subject, predicate, object, graph but here there are only 3 arguments given, I believe it's missing the object/value.

Reply:
pass `undefined` (or if that doesn't work, `null`) as the fourth argument